assets/js/components/HealthSummary/HomeHealthSummary.test.jsx

```diff
@@ -53,6 +60,44 @@ describe('HomeHealthSummary component', () => {
         .getAttribute('href')
     ).toContain(`/sap_systems/${id}`);
   });
+
+  it('should have a clickable PACEMAKER CLUSTER icon with link to the belonging cluster when available', () => {
+    const [StatefulHomeHealthSummary] = withState(
+      <HomeHealthSummary />,
+      initialState
+    );
+    const { container } = renderWithRouter(StatefulHomeHealthSummary);
+
+    const availablePacemakerClusterSelector =
+      ':nth-child(1) > :nth-child(4) > a';
+
+    const [{ clusterId }] = homeHealthSummaryActionPayload;
+
+    expect(
+      container
+        .querySelector(availablePacemakerClusterSelector)
+        .getAttribute('href')
+    ).toContain(`/clusters/${clusterId}`);
+
+    expect(
+      container.querySelector(availablePacemakerClusterSelector).style
+        .pointerEvents
+    ).toBe('auto');
+
+    const unavailablePacemakerClusterSelector =
+      ':nth-child(4) > :nth-child(4) > a';
+
+    expect(
+      container
+        .querySelector(unavailablePacemakerClusterSelector)
+        .getAttribute('href')
+    ).toContain(`/clusters/null`);
+
+    expect(
+      container.querySelector(unavailablePacemakerClusterSelector).style
+        .pointerEvents
+    ).toBe('none');
+  });
```

Comment:
Actually I would expect no href to be present if the cluster id is null :sweat_smile: